Cloud Application Manager/Getting Started/self-register-instances.md

```diff
@@ -0,0 +1,100 @@
+{{{ "title": "Self-Registering Virtual Machines",
+"date": "09-04-2019",
+"author": "Denes Pal and Guillermo Sanchez",
+"attachments": [],
+"contentIsHTML": false,
+"keywords": ["cam","self-register", "register", "auto register", "provider"]
+}}}
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Supported provider types](#supported-provider-types)
+* [Supported instance types](#supported-instance-types)
+* [Enabling the self-registration feature for your organization](#enabling-the-self-registration-feature-for-your-organization)
+* [Enabling self-registration on a provider](#enabling-self-registration-on-a-provider)
+* [Obtaining the self-registration script](#obtaining-the-self-registration-script)
+* [Managing self-registered instances](#managing-self-registered-instances)
+* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+
+### Overview
+
+The self-Registering feature allow virtual machines that have been deployed outside of Cloud Application Manager to register themselves automatically into a provider in Cloud Application Manager as imported instances.
+
+The self-registration process is triggered by a command snippet available through the self-register enabled provider details page that has to be attached to the virtual machine first startup sequence. This snippet downloads and installs the Cloud Application Manager agent and registers the instance automatically into Cloud Application Manager.
+
+From there on, Cloud Application manager can be used to manage the lifecycle of the imported instance, as with any other deployed or manually registered instance.
+
+### Audience
+
+All Cloud Application Manager users who want to get their virtual machines deployed directly to the underlying provider or with any other deployment management tool to be automatically registered in Cloud Application Manager.
+
+### Prerequisites
+
+* Access to Cloud Application Manager [Management site](https://account.cam.ctl.io/#/providers?type=Amazon-Web-Services).
+* The user must have access to an already configured [provider](../Core Concepts/providers.md) in Cloud Application Manager, that supports the self-registration of virtual machines.
+* Access to the underlying cloud provider account with authorization to deploy virtual infrastructure.
+
+### Supported provider types
+
+These are the Provider types supporting self-registration of virtual machines:
+
+* Amazon AWS/EC2
+* CenturyLink Cloud
+* CenturyLink Private Cloud on VmWare Cloud Foundation
+* Google GCP/GCE
+* VmWare vCloud Director
```

Comment:
Same, I think this is still in progress

Reply:
Ok, I removed these two as well as CPC on vCF for now (we need to remember to add them afterwards whenever they are available).